bugbug/repository.py

```diff
@@ -130,14 +133,19 @@ def download_commits(repo_dir, date_from):
 
     # Total previous number of commits by the author.
     total_commits_by_author = defaultdict(int)
+    total_reviews_by_reviewer = defaultdict(int)
     # Previous commits by the author, in a 90 days window.
     commits_by_author = defaultdict(list)
 
     global author_experience
+    global reviewer_experience
     global author_experience_90_days
+
     for commit in commits:
         author_experience[commit] = total_commits_by_author[commit.author]
+        reviewer_experience[commit] = total_reviews_by_reviewer[commit.reviewers]
```

Comment:
You are using a list of reviewers here :smile:

Reply:
`commit.reviewers` returns a byte string, (`b'name'`), not a list.

Reply:
https://github.com/mozilla/version-control-tools/blob/d5db0980ab004b3aada29afdcf798a8a95520240/hgext/mozext/__init__.py#L1062